### PR TITLE
Remove scda@main

### PR DESCRIPTION
**Pull request by gogonzo**
linked to #299

### CODE REVIEW

**Review by m7pr — APPROVED** [review]

File: R/FilteredDataCDISC.R [hunk 1]
```diff
@@ -18,9 +18,10 @@
 #' @examples
 #' library(scda)
```
Comment (m7pr) [inline R/FilteredDataCDISC.R:19]: @gogonzo should you remove `library(scda)` as well?

**Review by m7pr — DISMISSED** [review]

File: R/FilteredDataCDISC.R [hunk 2]
```diff
@@ -18,9 +18,18 @@
...
+#'   USUBJID = letters[1:10],
+#'   SEX = sample(c("F", "M"), 10, replace = TRUE)
+#' )
+#' ADTTE <- data.frame(
+#'   STUDYID = 1,
+#'   USUBJID = rep(letters[1:10], each = 3),
+#'   PARAMCD = rep(c("P1", "P2", "P3"), each = 10),
+#'   AVAL = runif(30)
+#' )
+#' formatters::var_labels(ADSL) <- colnames(ADSL)
```
Comment (gogonzo) [inline R/FilteredDataCDISC.R:32]: @pawelru looks good?

Comment (pawelru) [inline R/FilteredDataCDISC.R:32]: is this needed?

Comment (gogonzo) [reply R/FilteredDataCDISC.R]: nope, removed

**Review by pawelru — APPROVED** [review]
LGTM 👍

File: R/resolve_state.R [hunk 3]
```diff
@@ -62,7 +62,7 @@ resolve_state.default <- function(x) {
 #' @rdname resolve_state
 #' @keywords internal
 #' @export
-resolve_state.default_filter <- function(x, filter_state) { # nolint
+resolve_state.default_filter <- function(x) { # nolint
```
Comment (gogonzo) [inline R/resolve_state.R:65]: this was an obvious error on the main. `resolve_state` is never called with two arguments

Comment (pawelru) [reply R/resolve_state.R:65]: looks like our recent efforts of converting some notes to errors pays-off :)

**Pull request merged**